static/styles/settings.css

```diff
@@ -167,6 +167,20 @@ h3,
     cursor: not-allowed;
 }
 
+#deactivate_realm_personal_setting_container {
+    display: inline-block;
+    @media (max-width: 380px) {
+        margin-top: 18px;
```

Comment:
I think we can keep this margin as 20px to make it consistent with the margin between other inputs.

Reply:
done.